docs/tools-docs.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Walkthrough
+---
+
+## In a nutshell, RMRK tools:
+
+- GET complete state of all remarks right now
+- RETRIEVE remarks from extrinsic block call
+- VALIDATE remarks
+- CREATE new remark strings ready to be sent as transactions
+
+This walkthrough of our Tools is based on the 1.0 spec. We will be releasing v2.0 docs whenever RMRK2.0 hits the shelves! For the best developer experience, we recommend developing with the most current **Mac operating system**.
+
+### Warning: Not a stroll in the park
+
+Getting RMRK Tools to run properly in local development is currently not a plug-and-play experience. You will need to be familiar with creating your first Substrate node using the Rust programming language; take 2 hours to get yourself jumpstarted [here](https://substrate.dev/tutorials).
```

Comment:
You can just download the Polkadot or Kusama binary and run it if on Ubuntu. The tutorial to do a compilation, though, is more like 10 minutes + 40 min compilation time.